packages/artifact/src/internal/upload/zip.ts

```diff
@@ -5,6 +5,8 @@ import {createReadStream} from 'fs'
 import {UploadZipSpecification} from './upload-zip-specification'
 import {getUploadChunkSize} from '../shared/config'
 
+export const DEFAULT_COMPRESSION_LEVEL = 6
```

Comment:
Nit, potentially you can put the default value into `packages/artifact/src/internal/shared/config.ts` but no biggie

Reply:
Since it's constant and only used for the zip method in this file, I'm just going to leave it here for now 👌